src/ast/ast_build_filter_tree.c

```diff
@@ -116,21 +118,7 @@ static FT_FilterNode *_convertUnaryOperator(const cypher_astnode_t *op_node) {
 	}
 }
 
-static FT_FilterNode *_convertApplyOperator(const cypher_astnode_t *op_node) {
-	return FilterTree_CreateExpressionFilter(AR_EXP_FromASTNode(op_node));
-}
-
-static FT_FilterNode *_convertTrueOperator() {
-	AR_ExpNode *exp = AR_EXP_NewConstOperandNode(SI_BoolVal(true));
-	return FilterTree_CreateExpressionFilter(exp);
-}
-
-static FT_FilterNode *_convertFalseOperator() {
-	AR_ExpNode *exp = AR_EXP_NewConstOperandNode(SI_BoolVal(false));
-	return FilterTree_CreateExpressionFilter(exp);
-}
-
-static FT_FilterNode *_convertIntegerOperator(const cypher_astnode_t *expr) {
```

Comment:
This is great!

Reply:
Long overdue! This file was unnecessarily complicated.